docs/sources/json.md

```diff
@@ -44,6 +44,31 @@ Note that:
 * For list we define a key with its index and they can nest other values.
 * All `Key`s are lowercase so this source ignores uppercase properties.
 
+### Magic `_self` key
+
+Conferer allows setting some key and at the same time more specifici keys at the same
+time.
```

Comment:
```suggestion

Conferer allows setting a value for a key and for more specific keys at the same time.
```

what about something like `subkeys of that key` instead of `more specific keys`?